.github/workflows/test-application.yaml

```diff
@@ -89,13 +89,13 @@ jobs:
               if: ${{ !matrix.create-project }}
               uses: ramsey/composer-install@v1
               with:
-                  composer-options: '--working-dir=${{ matrix.working-directory }}'
+                  composer-options: '--no-cache --working-dir=${{ matrix.working-directory }}'
```

Comment:
Just for completion: We are doing this because the `ramsey/composer-install` action caches and consequently installs an old dev version of the `sulu/sulu` dependency. This is especially annoying if we update the skeleton requires a newer `sulu/sulu` version than is installed. 

At the moment, the pipeline fails because this repository registers the newly added `SuluActivityBundle`, but the composer cache of the `ramsey/composer-install` action installs a `sulu/sulu` version that does not include that bundle yet: https://github.com/sulu/skeleton/runs/2528230627